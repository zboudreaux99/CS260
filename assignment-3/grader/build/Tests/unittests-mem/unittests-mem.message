Output line 0:
  <i>expected: </i>1..9 <span class='expected'>ok</span> 1 - testRLAddDiscrete ok 2 - testRLAddDuplicates ok 3 - testRLA
  <i>observed: </i>1..9 <span class='observed'>testReadingList</span>.cpp:52: error: Failed test # testReadingList.cpp:52: rl.size() 
