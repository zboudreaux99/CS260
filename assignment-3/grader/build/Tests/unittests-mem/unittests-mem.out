1..9
testReadingList.cpp:52: error: Failed test
# testReadingList.cpp:52: 	rl.size() isEqualTo(3)
# 	Expected: 3
# 	Observed: 0
# : 
not ok 1 - testRLAddDiscrete
testReadingList.cpp:91: error: Failed test
# testReadingList.cpp:91: 	rl.size() isEqualTo(3)
# 	Expected: 3
# 	Observed: 0
# : 
not ok 2 - testRLAddDuplicates
testReadingList.cpp:156: error: Failed test
# testReadingList.cpp:156: 	rl2.size() isEqualTo(2)
# 	Expected: 2
# 	Observed: 0
# : 
not ok 3 - testRLAssign
ok 4 - testRLConstructor
testReadingList.cpp:189: error: Failed test
# testReadingList.cpp:189: 	rl2.size() isEqualTo(2)
# 	Expected: 2
# 	Observed: 0
# : 
not ok 5 - testRLCopy
ok 6 - testRLDefaultConstructor
testReadingList.cpp:219: error: Failed test
# testReadingList.cpp:219: 	rl.size() isEqualTo(3)
# 	Expected: 3
# 	Observed: 0
# : 
not ok 7 - testRLInput
testReadingList.cpp:121: error: Failed test
# testReadingList.cpp:121: 	rl.size() isEqualTo(2)
# 	Expected: 2
# 	Observed: 0
# : 
not ok 8 - testRLRemove
testReadingList.cpp:239: error: Failed test
# testReadingList.cpp:239: 	leaked isEqualTo(0)
# 	Expected: 0
# 	Observed: 549
# : 
not ok 9 - zzMemoryCheck
# UnitTest: passed 2 out of 9 tests, for a success rate of 22.2%

--- end of output ---
--- std error ---
Leaked memory: 549 blocks
